<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoTeacherExamDialog</class>
 <widget class="QDialog" name="DoTeacherExamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>133</y>
     <width>45</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>题目:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>答案：</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>63</y>
     <width>256</width>
     <height>213</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QTextEdit" name="topicTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerLineEdit"/>
  </widget>
  <widget class="QLabel" name="numberLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>13</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>第i题        分数：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmPushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
